.github/workflows/feature-branch-auto-merge.yml

```diff
@@ -0,0 +1,34 @@
+name: Feature branch auto merge
+
+permissions: read-all
+
+on:
+  push:
+    branches:
+      - main  # Set a branch to deploy
+      
+env:
+
+  MY_REPO: https://GoogleCloudPlatform:${{secrets.GITHUB_TOKEN}}@github.com/GoogleCloudPlatform/magic-modules.git
```

Comment:
I think this shouldn't be necessary since the feature branch won't be in a separate repo - does that seem correct?